aces/Loris
# [Documentation] Delete file dataquery.md

This file belongs as a readme - a new readme file has been created already - PR #5206 . This file should be replaced with help text for the front-end user.

## Brief summary of changes


#### Testing instructions

1. ensure file has been deleted properly
2. ensure readme file is located in correct place

#### Links to related tickets (GitHub, Redmine, ...)

part of issue [https://github.com/aces/Loris/issues/5196](5196)
